assets/locales/de/translation.json

```diff
@@ -113,6 +113,7 @@
   "testingbotCredentialsRequired": "TestingBot key and secret are required!",
   "Perfecto SecurityToken is required": "Perfecto SecurityToken is required",
   "browserstackCredentialsRequired": "Browserstack username and access key are required!",
+  "lambdatestCredentialsRequired": "Lambdatest username and access key are required!",
```

Comment:
Could you add change only in english file? (`/en`)
https://github.com/appium/appium-inspector#localization

Reply:
this is done in the latest commit, thank you .